### PR TITLE
[MU4] Join/split measure(s)

### PR DESCRIPTION
**Pull request by Eism**
Peek 2020-11-05 13-10

### CODE REVIEW

File: mu4/notation/internal/notationinteraction.cpp [hunk 1]
```diff
@@ -1910,6 +1910,46 @@ mu::async::Notification NotationInteraction::textEditingChanged() const
     return m_textEditingChanged;
 }
 
+void NotationInteraction::splitMeasure()
+{
+    Element* selectedElnement = m_selection->element();
```
Comment (Howard-C) [inline mu4/notation/internal/notationinteraction.cpp:1915]: Typo, `selectedElement`.

**Comment by igorkorsukov** [comment]
needs rebase

**Comment by Eism** [comment]
@igorkorsukov, fixed

File: mu4/notation/internal/notationselectionrange.cpp [hunk 2]
```diff
@@ -258,3 +268,17 @@ int NotationSelectionRange::sectionElementsMinY(const NotationSelectionRange::Ra
 
     return minY;
 }
+
+NotationSelectionRange::MeasureRange NotationSelectionRange::measureRange() const
+{
+    const Ms::Selection& selection = score()->selection();
+    Measure* startMeasure;
```
Comment (RomanPudashkin) [inline mu4/notation/internal/notationselectionrange.cpp:275]: Please, assign nullptr to those pointers

File: mu4/notation/internal/notationactioncontroller.cpp [hunk 3]
```diff
@@ -19,6 +19,7 @@
 #include "notationactioncontroller.h"
 
 #include <QPoint>
+#include "libmscore/measure.h"
```
Comment (RomanPudashkin) [inline mu4/notation/internal/notationactioncontroller.cpp:22]: Please, remove this header

File: mu4/notation/qml/MuseScore/NotationScene/SelectMeasuresCountDialog.qml [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+            }
+
+            Row {
+                Layout.preferredHeight: childrenRect.height
+                Layout.alignment: Qt.AlignRight | Qt.AlignBottom
+
+                spacing: 12
+
+                FlatButton {
+                    text: qsTrc("notation", "Cancel")
```
Comment (RomanPudashkin) [inline mu4/notation/qml/MuseScore/NotationScene/SelectMeasuresCountDialog.qml:93]: You should replace "notation" with "global"

**Review by RomanPudashkin — APPROVED** [review]

**Review by igorkorsukov — APPROVED** [review]

**Pull request merged**